README.md

```diff
@@ -1,69 +1,60 @@
+  * liberal open source license: allows integration with any app or library, requires only attribution
+  * very high code quality standards: >97% unit test coverage, performance regression tests...
+  * extensive API documentation & tutorials: more lines of documentation than lines of code
 
-To build the tests and API documentation, you also need:
+## Dependencies
 
-|                   | requirement                                          | version  |
-|-------------------|------------------------------------------------------|----------|
-|**build system**   | [cmake](https://cmake.org)                           | ≥ 3.4    |
-|**test system**    | [GoogleTest](https://github.com/google/googletest)   | ≥ 1.8    |
-|**doc system**     | [Doxygen](https://github.com/doxygen/doxygen)        | ≥ 1.8    |
+|                   | requirement                                          | version  | comment                                    |
+|-------------------|------------------------------------------------------|----------|--------------------------------------------|
+|**compiler**       | [GCC](http://gcc.gnu.org)                            | ≥ 7      | no other compiler is currently supported!  |
+|**build system**   | [cmake](https://cmake.org)                           | ≥ 3.4    | optional, but recommended                  |
+|**required libs**  | [SDSL](https://github.com/xxsds/sdsl-lite)           | ≥ 3      |                                            |
+|                   | [Range-V3](https://github.com/ericniebler/range-v3)  | ≥ 1.0    |                                            |
+|**optional libs**  | [Cereal](https://github.com/USCiLab/cereal)          | ≥ 1.2.3  | required for serialisation and CTD support |
+|                   | [zlib](https://github.com/madler/zlib)               | ≥ 1.2    | required for `*.gz` and`.bam` file support |
+|                   | [bzip2](http://www.bzip2.org)                        | ≥ 1.0    | required for `*.bz2` file support          |
 
 ## Using
 
-To use SeqAn3, clone this repository including it's submodules:
+We recommend that you use CMake to build your project:
 
-```sh
-git clone --recursive https://github.com/seqan/seqan3.git
-```
+  * ["Setup-Tutorial"](http://docs.seqan.de/seqan/3.0.0-master-user/setup.html)
+  * Using CMake guarantees that all optional dependencies are automatically detected and activated.
 
-SeqAn3 is still a header-only library so it is sufficient to add the `include` folder to your include path. When building make sure that you add the required parameters:
+Quick-Setup without CMake:
 
+  * Clone the repository with submodules: `git clone --recursive https://github.com/seqan/seqan3.git`
+  * Add the following to your compiler invocation:
+    * the include directories of SeqAn and its dependencies
+    * C++17 mode with concepts support
+    * Macros indicating the presence of ZLib and BZip2 (if available)
```

Comment:
```suggestion
    * Macros indicating the presence of zlib and bzip2 (if needed)
```

Reply:
I'd either change it to `needed` since it's optional. Or remove it since `indicating the presence` and `if available` is kinda redundant

Reply:
Hm, but you only set the Macros and indicate the presen **if** they are available. You can also set the macros to indicate the presence if they are not available which will break everything. 

I have tried to make it clearer.